File: relaciones_poo/Empleado.java

package relaciones_poo;

public class Empleado {
    
    public String Cargo;
    public String Nombre;
        
    public Empleado(String Nombre, String Cargo){
        this.Nombre = Nombre;
        this.Cargo = Cargo;
    }
        
    public String getNombre(){
        return Nombre;
    }
    
    public String getCargo(){
        return Cargo;
    }
        
    @Override
        
    public String toString(){
        return Nombre;
    }
}

File: Empleado.py
class Empleado:
    def __init__(self, Nombre, Cargo):
        self.Nombre = Nombre
        self.Cargo = Cargo
        
class Tarea:
    def __init__(self, Nombre, Responsable):
        self.Nombre = Nombre
        self.Responsable = Responsable
        
Empleado = Empleado("Juan Diego","Ing de Soporte")
Tarea = Tarea("Soporte Nivel 3 con su usuario","Juan Diego")
print(Empleado.Nombre, "-", "Tarea.Nombre")

Empleado = Empleado("Juan Camilo","Ing ML")
Tarea = Tarea("General Datasets","Juan Camilo")
print(Empleado.Nombre, "-", "Tarea.Nombre")